tests/testthat/test-accessors.R

```diff
@@ -0,0 +1,11 @@
+test_that("`[[` allows retrieval of objects from cache", {
+  x <- LRUcache(1)
+  x$set("a", 10)
+  expect_equal(x[["a"]], 10)
+})
+
+test_that("`[[` allows assignment of objects to cache", {
+  y <- LRUcache(1)
+  y[["b"]] <- 20
+  expect_equal(y$get("b"), 20)
+})
```

Comment:
It's great that tests work locally, but I'm pretty sure that those methods need to be `@export`'ed